app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -575,8 +572,6 @@ class GleanMetricsService(private val context: Context) : MetricsService {
     }
 
     override fun stop() {
-        gleanInitializer.cancel()
```

Comment:
Since we're now avoiding the job/coroutine scope, deleting this portion is fine. The queue doesn't support cancelling them, so we will end up running them even if app is backgrounded

Mike had brought up an interesting point:  "If we never draw the frame that these get launched from, will they launch when the app is reopened?" I verified that tasks run in the background and are not triggered again when we go navigate back into the app unless we force a cold start (in which case we should be entering this block again)

cc: @mcomella